.github/workflows/release.yaml

```diff
@@ -65,12 +51,14 @@ jobs:
     runs-on: macos-12
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - run: rm rust-toolchain.toml
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          target: x86_64-apple-darwin,aarch64-apple-darwin
+          toolchain: nightly-2023-01-31
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
-          profile: minimal
-          toolchain: nightly
-          target: aarch64-apple-darwin
-      - uses: Swatinem/rust-cache@v2
+          target: aarch64-apple-darwin,aarch64-apple-darwin
```

Comment:
This looks a bit like a duplicate, is this on purpose?

Reply:
nice catch!